lib/workload/stateful/token_service/register_service_user.py

```diff
@@ -0,0 +1,48 @@
+"""
+Usage:
+    export AWS_PROFILE=umccr-dev-admin
+    python register_service_user.py
+"""
+import dataclasses
+import json
+
+import boto3
+
+from token_service.cognitor import CognitoTokenService, ServiceUserDto
+
+ssm_client = boto3.client('ssm')
+
+
+def get_param(param_name):
+    return ssm_client.get_parameter(Name=param_name, WithDecryption=True)['Parameter']['Value']
+
+
+if __name__ == '__main__':
+    # We need to know Cognito User Pool ID and, the associated App Client ID.
+    # In this case, we are using Portal App Client ID to streamline with existing UI frontend.
+    # These SSM param lookup keys should be in-sync with what goes in the deployment `component.ts`.
+    cognitor = CognitoTokenService(
+        user_pool_id=get_param('/data_portal/client/cog_user_pool_id'),
+        user_pool_app_client_id=get_param('/data_portal/client/data2/cog_app_client_id_stage')
+    )
+
+    user_dto = ServiceUserDto(
+        username='orcabus.api',
+        email='services+orcabus.api@umccr.org',
```

Comment:
Umm. Good point. This script is not part the of the deployment nor application scope. It is just an admin script live outside both scopes.

It can be something admin user enter into Cognito User Pool with however values that admin see fit here. 

Let me see whether I can refactor this into upstream Cognito better to distinct the point - the fact that some admin process (or upstream Cognito setup) has to do this part. Rather than down under here at application side. i.e. moving it into somewhere in Cognito TF tree - https://github.com/umccr/infrastructure/tree/master/terraform/stacks/cognito_aai

Reply:
Oh, there is user resource avail in TF. Let me explore that. We can replace this service user provisioning script with it at TF side.

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/cognito_user

Reply:
Resolved and now handled by https://github.com/umccr/infrastructure/pull/412

Hence, I will remove this `register_service_user.py` script method to avoid confusion.

Reply:
Gone. Updated README to reflect the amended service user registration arrangement.